test/src/util.js

```diff
@@ -15,15 +18,7 @@ export function uniqueId() {
  * Authorizes the user account.
  */
 export async function authorize() {
```

Comment:
```suggestion
export async function login() {
```
This was originally named after the Girder 3 feature, which is no longer relevant.

Reply:
https://github.com/dandi/dandiarchive/pull/848/commits/ee1cc5efc656756f0f9eb55c11a47c27ca52cd55